aten/src/ATen/cuda/LegacyTHFunctionsCUDA.cpp

```diff
@@ -163,6 +163,12 @@ std::tuple<Tensor,Tensor> _th_sort_stable(const Tensor & self, c10::optional<boo
             THCudaHalfTensor_sort(globalContext().getTHCState(), values_, indices_, self_, dim, descending);
             break;
         }
+	case ScalarType::BFloat16: {
+            auto self_ = checked_dense_tensor_unwrap(self, "self", 1, "_th_sort", false, DeviceType::CUDA, dispatch_scalar_type);
+            THCudaBFloat16Tensor_sort(globalContext().getTHCState(), values_, indices_, self_, dim, descending);
+            break;
+
+	}
```

Comment:
Does sort support bfloat16? I don't think it does. Because cub radix sort does not support bfloat16.

Reply:
So one issue I noticed that is it looks like the old legacy sort in THC supported `BFloat16` (or at least claimed to)? Because registering `topk` without dispatching for `BFloat16` triggers errors in CI...

Reply:
It seems that someone added bfloat16 support for topk, but that person only added it for the <2048 path, but not the sort path? This sounds like a bug.

Reply:
Yes, although I am not sure if removing the limited support for BFloat16 is the right way to go either...

Reply:
I think the right thing to do is to open an issue for this, mentioning that the bfloat16 support for topk is not complete.